src/configuration/prima.rs

```diff
@@ -131,12 +187,44 @@ impl FromStr for Country {
 
 impl Display for Country {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        match self {
-            Country::Common => write!(f, "common"),
-            Country::It => write!(f, "it"),
-            Country::Es => write!(f, "es"),
-            Country::Uk => write!(f, "uk"),
+        f.write_str(self.as_str())
+    }
+}
+
+#[cfg_attr(docsrs, doc(cfg(feature = "serde")))]
+#[cfg(feature = "serde")]
+impl serde::Serialize for Country {
```

Comment:
I think for this and for `Environment` you can derive `Serialize` and `Deserialize` putting on top of the enum the `#[serde(rename_all = "lowercase")]` (or `snake_case`)

Reply:
Good spot... only trouble is that it slightly clashes with the non-serde `as_str` and `FromStr` implementations (whilst they're equal to the serde rename_all=lowercase at the moment we might add something later that won't be), and it gets rid of the nice feature-flag tag on docs.rs :(

![image](https://user-images.githubusercontent.com/14863743/204568408-b682eb58-9b13-4646-86f1-9de3e025c130.png)

Reply:
:+1: